Responsive breadcrumbs

ezequielc opened the pull request:
Breadcrumbs are responsive based on the width of parent container.
It will always show root, current and parent folders. It will hide in between folders if there is not enough space. Also, every folder name has a limit of 20 chars.
@alex-deaconu Please review.

alex-deaconu commented on web/partials/storage/files-list.html:
```diff
@@ -62,7 +62,7 @@
         ng-click="fileClick(file);"
         ng-class="{'active': file.isChecked, 'blocked-file': file.isThrottled, 'back-btn': file.currentFolder, 'no-select-row': !storageFactory.canSelect(file), 'disabled-row': storageFactory.isDisabled(file) }"
-        ng-repeat="file in filesDetails.files | filter:search.query | orderBy:orderByAttribute:reverseSort track by $index" ng-if="!storageFactory.fileIsTrash(file) || storageFactory.storageFull">
+        ng-repeat="file in filesDetails.files | filter:search.query | orderBy:orderByAttribute:reverseSort track by $index" ng-if="file.name !== storageFactory.folderPath && (!storageFactory.fileIsTrash(file) || storageFactory.storageFull)">
```
@ezequielc wondering if instead of filtering out the `folderPath` folder we should just prevent it from being created here: https://github.com/Rise-Vision/rise-vision-apps/blob/master/web/scripts/storage/services/svc-files-factory.js#L88. Thanks.

ezequielc replied:
@alex-deaconu I thought about it but when I read the comment about Trash (https://github.com/Rise-Vision/rise-vision-apps/blob/master/web/scripts/storage/services/svc-files-factory.js#L104) I didn't try. But it is a good suggestion. I will try and see how it works. Thanks

alex-deaconu commented:
@ezequielc looks good 👍 . Just one suggestion in the code. Thanks.

ezequielc commented:
@alex-deaconu I removed the code to add current folder to the list, so it will not appear anymore. Also, Storage Home and Selector layouts have been updated. I squashed the code as we had too many commits here. We can merge and we are OK to release. Please review. Thanks!

alex-deaconu commented:
@ezequielc looks good 👍 . Thanks for updating!
